LondonBritishLibrary/orient/BLorient11601.xml

```diff
@@ -0,0 +1,305 @@
+                                </msItem>
+                                <msItem xml:id="p1_i1.6">
+                                    <locus from="22va" to="24va"/>
+                                    <title ref="LIT1694Jonah"/>
+                                    <note>In the upper margin of <locus target="#22v"/>: <foreign xml:lang="gez">አመ፡ ፳ወ፭ለመስከረም፡ ዮናስ።</foreign></note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.7">
+                                    <locus from="24va" to="26va"/>
+                                    <title ref="LIT2057Bookof"/>
+                                    <note>In the upper margin of <locus target="#24v"/>: <foreign xml:lang="gez">አመ፡ ፭ለታኅሣሥ፡ ናሆም።</foreign></note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.8">
+                                    <locus from="26va" to="29ra"/>
+                                    <title ref="LIT1567Bookof"/>
+                                    <note>In the upper margin of <locus target="#26v"/>: <foreign xml:lang="gez">አመ፡ ፫ለኅዳር፡ እንባቆም።</foreign></note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.9">
+                                    <locus from="29ra" to="31va"/>
+                                    <title ref="LIT3148Zephan"/>
+                                    <note>In the upper margin of <locus target="#29r"/>: <foreign xml:lang="gez">አመ፡ ፬ለሐምሌ፡ ሶፎንያስ።</foreign></note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.10">
+                                    <locus from="31va" to="33rb"/>
+                                    <title ref="LIT3149Haggai"/>
+                                    <note>In the upper margin of <locus target="#31v"/>: <foreign xml:lang="gez">አመ፡ ፳ለታኅሣሥ፡ ።</foreign></note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.11">
+                                    <locus from="33rb" to="42va"/>
+                                    <title ref="LIT3150Zechar"/>
+                                    <note>In the upper margin of <locus target="#33r"/>: <foreign xml:lang="gez">አመ፡ ፲ወ፭፡ ለየካቲት፡ ዘካርያስ።</foreign></note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.12">
+                                    <locus from="42va" to="45va"/>
+                                    <title ref="LIT3151Malachi"/>
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="p1_i2">
+                                <locus from="46ra" to="75rb"/>
+                                <title ref="LIT1109Anqasa"/>
+                                <textLang mainLang="gez"/>
```

Comment:
I would include a note that the witness was used in the edition of van Donzel (since the copies of the book are rare); I would also include this book in the bibliography, because it contains a description of the manuscript.